src/compiler/utilities.ts

```diff
@@ -1,5 +1,6 @@
 /// <reference path="binder.ts" />
 
+/** @internal */

```

Comment:
Just being picky, could we use same syntax between this. We use both /** @internal */ and /* @internal */